lxqt-admin-user/mainwindow.cpp

```diff
@@ -40,6 +41,7 @@ MainWindow::MainWindow():
     connect(ui.actionChangePasswd, SIGNAL(triggered(bool)), SLOT(onChangePasswd()));
     connect(ui.actionRefresh, SIGNAL(triggered(bool)), SLOT(reload()));
 
+    connect(ui.tabWidget,SIGNAL(currentChanged(int)),SLOT(onTabChange(int)));
```

Comment:
I did  that because Q_OS_* is not usefull in moc context. I changed that by using a cmake variable system_detect.h.in instead. A little Cmake hack. feel free to suggest a better solution.